src/mapillary/interface.py

```diff
@@ -30,6 +30,36 @@
 import mapillary.controller.detection as detection
 import mapillary.controller.save as save
 
+def international_dateline(bbox):
+    if bbox['west']>0 and bbox['east']<0:
+        return True
+    return False
+
+def check_bbox_validity(bbox):
+    # longitude check
+    if bbox['west'] < 180 or bbox['east'] > 180:
+        return False
+    # lattitude check
+    elif bbox['north'] > 90 or bbox['south'] < 90:
+        return False
+    # longitude validity check
+    elif bbox['west']>bbox['east'] :
+        # extra check for international dateline
+        # it could either be an error or cross an internaitonal dateline
+        # hence if it is passing the dateline, return True
+        if international_dateline(bbox):
+            new_east = bbox['east'] + 360
+            bbox['east'] = new_east
+            return bbox
+        return False
+    # lattitude validitiy check
+    elif bbox['north']<bbox['south']:
+        return False
+    elif bbox['north']==bbox['south'] and bbox['west']==bbox['east']:        
+        # checking for equal values to avoid flat box 
+        return False
+    else:
+        return True
```

Comment:
These functions should be moved to `utils/verify.py`, and then imported from `verify.py` to `interface.py`.

Reply:
resolved

Reply:
:partying_face: